### PR TITLE
Newhome layout (made it responsive)

### PR DESCRIPTION
**Pull request by Prakhar-Shankar**
Issue #1617 

Now in mobile view we are following this sequence - 
1) Navbar
2) Report Section 
3) Reported Bugs
4) Leaderboard

**Mobile View**

image

image

image

**Desktop view**

image

### CODE REVIEW

File: website/templates/new_home.html [hunk 1]
```diff
@@ -65,7 +94,7 @@
             });
         }
     </script>
-    <div class="col-md-12">
+    <div class="col-md-12 ">
```
Comment (DonnieBLT) [inline website/templates/new_home.html:97]: Why a space here?

File: website/templates/new_home.html [hunk 2]
```diff
@@ -6,19 +6,48 @@
...
+        <div class="w-full mt-4  sm:w-[20%] sm:mt-0 sm:order-2 ">
+            <!-- Leaderboard Widget -->
+            <div class="mx-4 mr-10 ">
+                {% include '_leaderboard_widget.html' %}
+            </div>
+        </div>
     </div>
     
+    
+        <!-- <section class="hero-bugs-container grid grid-rows-2 gap-[25px] mx-[4%] my-[2.5rem] sm:max-w-screen-3xsm">
```
Comment (DonnieBLT) [inline website/templates/new_home.html:37]: Can remove this?

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
A couple tweaks

File: website/templates/new_home.html (hunk 2 above)
Comment (Prakhar-Shankar) [reply website/templates/new_home.html:37]: sure doing it right now.

**Comment by Prakhar-Shankar** [comment]
> A couple tweaks

@DonnieBLT  I have made the changes sir.

**Review by DonnieBLT — APPROVED** [review]

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**